.husky/set-node.sh

```
@@ -285,50 +279,50 @@ check_latest_pnpm_installed() {
+    v2_major=$(printf "%s" "$2" | cut -d. -f1)
+    v2_minor=$(printf "%s" "$2" | cut -d. -f2)
+    v2_patch=$(printf "%s" "$2" | cut -d. -f3)
 
     # 값이 없는 경우 0으로 설정
     : "${v1_major:=0}" "${v1_minor:=0}" "${v1_patch:=0}"
     : "${v2_major:=0}" "${v2_minor:=0}" "${v2_patch:=0}"
 
     # 메이저 버전 비교
     if [ "$v1_major" -lt "$v2_major" ]; then
-      echo "-1"
+      printf "-1"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, printf -1 is undefined.

(SC3045)